http/src/main/resources/reference.conf

```diff
@@ -194,7 +194,7 @@ azure {
         type = "CustomScript",
         version = "2.1",
         minor-version-auto-upgrade = true,
-        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/8dc8a0171fa4267197305eddddafb9dda8ce97fa/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
+        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/IA-3957-include-additional-create-parameters/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
```

Comment:
What's best practice for keeping this line correct?
I know it shouldn't stay as-is. 
Do I find a permalink for the file that matches this PR? (Seems like historically this is what we do.)
Do we want to keep this reference.conf pointing to the most up-to-date version from dev?

Reply:
Note - ConfigReaderSpec and WsmCodecSpec won't succeed as-is, they'll need updating to whatever we decide with this.

Reply:
My vote is to do the permalink from this feature branch only. It is easier to do here, and will be updated next time a dev makes a change to that script